Rakefile

```diff
@@ -26,6 +26,7 @@ RSpec::Core::RakeTask.new(:spec) do |spec|
   spec.rspec_opts = def_spec_options
   spec.rspec_opts << ' --tag ~integration'
   spec.rspec_opts << ' --tag ~system'
+  spec.rspec_opts << ' --tag ~integration_i3s'
```

Comment:
It won't be better to use two tags, instead creating one tag for each specific combination?